.peoplemap

```diff
@@ -41,37 +47,82 @@ Ken Dreyer <kdreyer@redhat.com> Ken Dreyer <ken.dreyer@inktank.com>
+Matthew Vernon <mv3@sanger.ac.uk> Matthew Vernon <mvernon@wikimedia.org>
+Marc Schoechlin <marc.schoechlin@vico-research.com> Marc Schoechlin <ms@256bit.org>
+Mark Kogan <mkogan@ibm.com> <mkogan@redhat.com>
+Mark Nelson <mark.nelson@clyso.com> <mnelson@redhat.com> <mark.nelson@inktank.com> <mark.nelson@dreamhost.com>
+Min Chen <chenmin@xsky.com> Min Chen <minchen@ubuntukylin.com>
+Ming Hao Cong <minghaocong@TENCENT64.site> Ming Hao Cong <minghaocong@tencent.com>
+Misono Tomohiro <misono.tm@gmail.com> Misono Tomohiro <misono.tomohiro@jp.fujitsu.com>
+Mitsumasa KONDO <kondo.mitsumasa@gmail.com> Mitsumasa KONDO <mitsu-ko@MitsumasanoMacBook-Pro.local>
+Mykola Golub <mykola.golub@clyso.com> <mgolub@suse.com> <mgolub@mirantis.com> <mgolub@suse.com> <to.my.trociny@gmail.com>
+Nathan Cutler <ncutler@suse.com> Nathan Cutler <presnypreklad@gmail.com>
 Neeraj Pratap Singh <Neeraj.Pratap.Singh1@ibm.com> <neesingh@redhat.com>
 Neil Levine <nlevine@redhat.com> Neil Levine <neil.levine@inktank.com>
+Nick Erdmann <n.erdmann@itc.rwth-aachen.de> Nick Erdmann <n@nirf.de>
+Ning Yao <yaoning@ruijie.com.cn> Ning Yao <yaoning@unitedstack.com>
 Noah Watkins <nwatkins@redhat.com> Noah Watkins <noah.watkins@inktank.com>
+Orit Wasserman <owasserm@ibm.com> <owasserm@redhat.com>
+Pan Liu <pan.liu@istuary.com> <wanjun.lp@alibaba-inc.com>
 Patrick McGarry <pmcgarry@redhat.com> Patrick McGarry <patrick@inktank.com>
 Paul Cuzner <pcuzner@ibm.com> <pcuzner@redhat.com>
+Paul Reece <paul@servercloud.com> <paulreece42@gmail.com> <paul@reece.cc>
+Pavan Rallabhandi <pavan.rallabhandi@sandisk.com> <PRallabhandi@walmartlabs.com>
+Peng Zhang <zhangpeng@vclusters.com> <zphj1987@gmail.com>
 Piotr Dałek <piotr.dalek@corp.ovh.com> Piotr Dałek <git@predictor.org.pl> Piotr Dałek <piotr.dalek@ts.fujitsu.com>
+Ponnuvel Palaniyappan <pponnuvel@gmail.com> <ponnuvel.palaniyappan@canonical.com>
 Radoslaw Zarzynski <rzarzyns@redhat.com> <rzarzynski@mirantis.com>
-Robin H. Johnson <rjohnson@digitalocean.com> <robin.johnson@dreamhost.com>
+Rahul Dev Parashar <d.rahul29@gmail.com> <rahul.dev@flipkart.com>
+Ruben Kerkhof <ruben@42on.com> <ruben@rubenkerkhof.com>
+Robin H. Johnson <rjohnson@digitalocean.com> <robin.johnson@dreamhost.com> <robbat2@gentoo.org>
 Ross Turk <rturk@redhat.com> Ross Turk <ross.turk@inktank.com>
+Sachin Punadikar <sachin.punadikar@ibm.com> <sachin.punadikar@seagate.com>
 Sage Weil <sweil@redhat.com> Sage Weil <sage@inktank.com>
-Sebastian Wagner <sewagner@redhat.com> Sebastian Wagner <swagner@suse.com>
+Sebastian Wagner <swagner@redhat.com> Sebastian Wagner <swagner@suse.com>
 Sahid Orentino Ferdjaoui <sahid.ferdjaoui@redhat.com> Sahid Orentino Ferdjaoui <sahid.ferdjaoui@cloudwatt.com>
 Samuel Just <sjust@redhat.com> Samuel Just <sam.just@inktank.com>
 Sandon Van Ness <svanness@redhat.com> Sandon Van Ness <sandon@inktank.com>
+Shang Ding <dingshang2013@163.com> <dingshang@xsky.com>
+Shilpa Jagannath <smanjara@redhat.com> <smanjara@li-b46676cc-2f05-11b2-a85c-9b92c8acacdf.ibm.com>
```

Comment:
Shilpa's ibm addr is smanjara@ibm.com. you can use .mailmap to map this strange ibm addr to that, then .peoplemap can use \<smanjara@redhat.com\> \<smanjara@ibm.com\> and .organizationmap can map \<smanjara@ibm.com\> to IBM

cc @smanjara